index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./styles/nav-bar.css">
+  <link rel="stylesheet" href="./styles/styles.css">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet">
+  <title>Global Training Courses - Home</title>
+</head>
+<body>
+  <header>
+    <nav class="nav-mobile">
+      <img class="menu-button" id="menu-button" src="./assets/hamburger-menu.svg" alt="menu">
+      <div class="menu" id="menu">
+        <div>
+          <img id="close-button" src="./assets/close.png" alt="close">
+        </div>
+        <ul>
+          <li><a href="about.html">About Us</a></li>
+          <li><a href="#topics">Topics</a></li>
+          <li><a href="#courses">Courses</a></li>
+          <li><a href="#">Log In</a></li>
+          <li><a href="#">Sign Up</a></li>
+        </ul>
+      </div>
+    </nav>
+    <nav class="nav-desktop">
+      <ul>
+        <li><a href="about.html">About Us</a></li>
+        <li><a href="#topics">Topics</a></li>
+        <li><a href="#courses">Courses</a></li>
+        <li><a href="#">Log In</a></li>
+        <li><a href="#">Sign Up</a></li>
+      </ul>
+    </nav>
+    <div class="title-container">
+      <h1>GLOBAL TRAINING COURSES</h1>
+      <p>A swiftly evolving business landscape requires an equally dynamic workforce. One-dimensional skill sets are no longer the norm, and not an option. To possess a blend of people, process and technology know-how makes a complete professional.
+      <br><br>Give your career a boost by mastering people and process expertise in areas like communication, critical thinking, finance, leadership, negotiation and problem-solving.</p>
+    </div>
+  </header>
+  <main>
+    <section class="topics" id="topics">
+      <h3 class="topics-title">Topics</h3>
+      <div class="border-topic"></div>
+      <div class="topics-container" id="topics-container">
+      </div>
+      <button>Join Global Training Courses</button>
+    </section>
+    <section class="courses-section" id="courses">
+      <h3>Courses</h3>
+      <div class="border-topic"></div>
+      <div class="courses-container" id="courses-container">
+      </div>
+      <button class="more-button" id="more-button">More</button>
+    </section>
```

Comment:
I really liked the way you implement your cards 💯  , however it would be great if you add an image for each course card
![image](https://user-images.githubusercontent.com/61790587/164914891-585d9c1b-e04b-4fa6-9222-197a5f698bd9.png)

Reply:
- [ ] Kindly add  images to your course cards, just as the previous reviewer has pointed out see the project design below
  
![feature](https://user-images.githubusercontent.com/66097371/164949659-0a42ffa3-103b-4c21-bf78-7550bc102d5c.png)